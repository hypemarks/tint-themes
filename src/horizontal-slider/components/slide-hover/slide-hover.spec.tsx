import { render } from '@testing-library/react';
import React from 'react';
import SlideHover from './slide-hover';

describe('Test <SlideHover /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(
      <SlideHover isMobile={false}>
        <div>content goes here</div>
      </SlideHover>
    );

    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
