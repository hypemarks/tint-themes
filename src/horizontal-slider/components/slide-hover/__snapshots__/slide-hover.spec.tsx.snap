// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <SlideHover /> Component should render snapshot 1`] = `
<DocumentFragment>
  <div
    class="slide-hover"
    style="opacity: 0;"
  >
    <div>
      content goes here
    </div>
  </div>
</DocumentFragment>
`;
